Tangem/Modules/Referral/ReferralView.swift

```diff
@@ -12,63 +12,195 @@ struct ReferralView: View {
     @ObservedObject var viewModel: ReferralViewModel
 
     var body: some View {
-        ScrollView {
-            VStack {
-                Assets.referralDude
-                    .resizable()
-                    .aspectRatio(contentMode: .fit)
-                    .padding(.horizontal, 40)
-                    .frame(maxHeight: 222)
-                
-                Text("referral_title".localized)
-                    .font(Fonts.Bold.title1)
-                    .multilineTextAlignment(.center)
-                    .lineLimit(2)
-                    .padding(.horizontal, 57)
-                    .padding(.top, 20)
-                    .padding(.bottom, 32)
-                
-                content
+        GeometryReader { geometry in
+            ScrollView {
+                VStack(spacing: 0) {
+                    Assets.referralDude
+                        .resizable()
+                        .aspectRatio(contentMode: .fit)
+                        .padding(.horizontal, 40)
+
+                    Text("referral_title".localized)
+                        .font(Fonts.Bold.title1)
+                        .multilineTextAlignment(.center)
+                        .lineLimit(2)
+                        .padding(.horizontal, 57)
+                        .padding(.top, 28)
+                        .padding(.bottom, 32)
+
+                    content
+                        .padding(.bottom, geometry.safeAreaInsets.bottom == 0 ? 10 : geometry.safeAreaInsets.bottom)
```

Comment:
```suggestion
                        .padding(.bottom, max(10, geometry.safeAreaInsets.bottom))
```
будет либо 10 либо safeArea

Reply:
заменил